app/src/common/metric-names.ts

```diff
@@ -0,0 +1,10 @@
+const server = 'jenkins-for-jira.forge.server';
+
+export const metricResolverEmitter = {
+	fetchJenkinsEventHandlerUrl: `${server}.fetchServer.emitted`,
+	connectJenkinsServer: `${server}.createServer.emitted`,
```

Comment:
curious why on the left hand side is "connect" while on the right hand side is "create"? Should we keep it uniform, wdyt?